Lib/posixpath.py

```diff
@@ -65,13 +69,17 @@ def isabs(s):
     """Test whether a path is absolute"""
     s = os.fspath(s)
     sep = _get_sep(s)
+    if _vxworks: #VxWorks paths dont always start with / and there is no good
+                 # way to find if a path is absolute. V7COR-3074, F7233
+        if not isinstance(s, str):
+            s = s.decode(sys.getdefaultencoding())
+        return bool(_vxwapi.isAbs(s))
```

Comment:
Again, you are doing the _vxworks test on every function call for every platform.  Not a huge deal but isabs() might be called a lot.  Instead, define a new isabs() on vxworks.